skcosmo/datasets/descr/degenerate_manifold.rst

```diff
@@ -0,0 +1,47 @@
+.. _degenerate_manifold:
+
+Degenerate manifold
+-------------------
+
+**Data Set Characteristics:**  
+
+    :Number of Instances: Each representation 162
+
+    :Number of Features: Each  representation 12
+
+The dataset contains two representations (SOAP power spectrum and bispectrum) of the two manifolds spanned by the carbon atoms of two times 81 methane structures [1]_.
+The SOAP power spectrum representation the two manifolds intersect creating a degenerate manifold/line for which the representation remains the same.
+In contrast for higher body order representations as the (SOAP) bispectrum the carbon atoms can be uniquely represented and do not create a degenerate manifold.
+Following the naming convention of Ref. [1]_ for each representation the first 81 samples correspond to the X minus manifold and the second 81 samples contain the X plus manifold 
+The representations were computed with [2]_ using the hyperparameters:
```

Comment:
Maybe add the commit version of librascal you used?

Reply:
Tthere is no version number, I could add the commit hash, or make an issue that this should be properly versioned as soon there is version.

Reply:
*or/and

Reply:
Yeah, I was thinking of the commit hash since there is not version on the librascal side yet.

Reply:
done